templates/etc/rstudio/rserver.conf.j2

```diff
@@ -4,3 +4,7 @@
 {% from "macros/ini_encode_macro.j2" import ini_encode with context -%}
 {{ ini_encode(rstudio_workbench_rserver_config) }}
 {% endif %}
+
+{% if rstudio_workbench_nfs_mount_path is defined and rstudio_workbench_nfs_mount_path|length %}
```

Comment:
What is the rationale behind using dedicated variable for this rather than putting it in `rstudio_workbench_rserver_config` and referencing it inside tasks as `rstudio_workbench_rserver_config['server-shared-storage-path']` ?